app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  before_validation :set_default_values
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
   validates :name, presence: true
   validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] Please add `:confirmable` because your application asks for confirmation of email.

- [ ] Please fix the issue in the video I recorded when running your app locally 

https://user-images.githubusercontent.com/75126481/224058605-0980f8fd-6ada-444b-9275-c68b75478364.mov